samtranslator/model/apigateway.py

```diff
@@ -121,6 +123,9 @@ def make_auto_deployable(
             hash_input.append(str(openapi_version))
         if domain:
             hash_input.append(json.dumps(domain))
+        if always_deploy:
+            # Using int so tests are a little more robust; don't think the Python spec defines default precision
+            hash_input.append(str(int(time.time())))
```

Comment:
do we still need `swagger`, `openapi_version` and `domain` as a part of hash if `AlwaysDeploy` is true?

Reply:
No, we just care that it changes. But due to how the existing code is written (chained `if`s, no separate function), doing it in another way would make the diff messy.

Reply:
Switched to assignment instead so future refactor is easier (no hash change).